.github/drafts/15938/15938_Audit_Logs_Outline.md

```diff
@@ -0,0 +1,108 @@
+# Audit logs
+
+Audit logs are a vital way to keep records of user activity related to app changes, platform settings, and sensitive data. With access to this information, you can ensure that you're able to easily follow up and address any unexpected or undesirable events.
+
+{Screenshot of audit logs page}
+
+---
+
+## Logged events
+
+(Should we include the technical names of the events? Or maybe one example event per type? i.e. `datasource.created` etc.)
+
+There are several types of events tracked in Appsmith's audit logs:
+
+- App-specific activity
+    - Changes to pages/widgets (`page.updated`)
+    - Datasource configuration(`datasource.updated`)
+    - Running queries(`query.executed`)
+    - Deleting apps
+
+- Admin settings changes
+    - Who changed certain settings (`instance_setting.updated`)
+    - Changes/new groups or roles (`group.updated`)
+
+- Sensitive information access
+    - Which users have accessed user data
+    - Which users have accessed billing data
+
+- User sign-up and logins
+    - New users
+    - Unsuccessful logins
+
+## Event details
```

Comment:
Are these event details? These are the contents of the audit log, correct?

Reply:
Yep! I went back-and-forth a couple of times on what to call this heading. It describes what data points are available as part of the logged event.

Reply:
We can call it **Audit Log Format** or **Audit Log Contents**

Reply:
@spark-c @AnaghHegde this section looks wrong to me.

This section is supposed to talk about the common properties of all events like timestamp, appsmith version, created at, user object and so on. Tomorrow when we add the IP address that will come here as well.

Reply:
Can you explain more what you mean? The section contains a table that will describe what each of those properties mean.

> Tomorrow when we add the IP address that will come here as well

Should I re-add the mentions of ipAddress in the table and JSON?

Reply:
@danciaclara Changed heading name to "Event contents" per request from docs team

Reply:
@spark-c we don't have to mention the IP Address in the table and JSON. We don't have that information today.

Reply:
@spark-c But these are not event contents. they are log contents. There will be confusion because **event** is one of the properties inside the log content. Hope that makes sense.